Holyhead-Harpies/Bangazon-Ruby-on-Rails-Web-Interface
# Jw fix seeds

## Status
**READY**

## Migrations
YES

## Description
Before, the seed file was throwing errors once our database rules were in place. The seed file was rewritten to seed data within the database restrictions.

## Related PRs
None


## Todos


## Deploy Notes
Fetch and checkout the branch
```
git fetch origin jw-fix-seeds
git checkout jw-fix-seeds
```
Reset the database
```
rails db:migrate:reset
```
Seed the database
```
rails db:seed
```
## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1.  Run through the steps to reset the database listed above
2. Open the development database in DBBrowser, all tables except images should be populated.
3. Check the Users table for the email address of one of the users. Login with that email address and password '123456'.  Data for that user should appear in all parts of the application.

## Impacted Areas in Application
List general components of the application that this PR will affect:
